features/Steps/UtilitySteps.php

```diff
@@ -37,4 +39,51 @@ public function iWaitSeconds(int $seconds): void
     {
         sleep($seconds);
     }
+
+    /**
+     * @Given /^I prevent duplicate creation$/
+     */
+    public function iPreventDuplicateCreation(): void
+    {
+        $this->switchedPreventDuplicateCreation = $this->changePreventDuplicateConfigFile(true);
+    }
+
+    /**
+     * @Then /^I allow duplicate creation$/
+     */
+    public function iAllowDuplicateCreation(): void
+    {
+        if (!$this->switchedPreventDuplicateCreation) {
+            return;
+        }
+
+        $this->changePreventDuplicateConfigFile(false);
+    }
+
+    private function changePreventDuplicateConfigFile(bool $bool): bool
+    {
+        $config = require 'config.php';
+
+        if ($config['prevent_duplicate_creation'] === $bool) {
+            return false;
+        }
+
+        $config['prevent_duplicate_creation'] = $bool;
+
+        file_put_contents('config.php', '<?php' . PHP_EOL . PHP_EOL . 'return ' . $this->shorthandVarExport($config) . ';');
+
+        return true;
+    }
+
+    // var_export uses old school array() style, this function converts this to the modern [] standard
+    private function shorthandVarExport(array $expression): string
```

Comment:
To avoid this you could also do a simple text replacement and don't convert to an array.

Replace `'prevent_duplicate_creation' => true` with `'prevent_duplicate_creation' => false` to turn the feature off.

 Replace `'prevent_duplicate_creation' => false` with `'prevent_duplicate_creation' => true` to turn the feature on.

Reply:
I didn't want to do a string conversion because that might be more vulnerable for slight differences in syntacs, for example somebody using double quotes for the property, or not using spaces.